src/lambda-powertools-layer.ts

```diff
@@ -32,6 +32,11 @@ export interface PowertoolsLayerProps {
    * The compatible architectures for the layer
    */
   readonly compatibleArchitectures?: lambda.Architecture[];
+
+  /**
+   * The Python version for Powertools for AWS Lambda (Python) V3.
+  */
+  readonly pythonVersion?: string;
```

Comment:
this needs to be narrowed down on the type level, so we can prevent typos and signal early that the version is correct. I'd suggest to pick a subset of `Runtime.PYTHON_XXX` as a type.

Reply:
I need to pass this string to the container, so, I'll need to cast this. Let me try.

Reply:
Done